packages/umi-rpc-web3js/src/createWeb3JsRpc.ts

```diff
@@ -384,7 +394,7 @@ function parseAccount(
     executable: account.executable,
     owner: fromWeb3JsPublicKey(account.owner),
     lamports: lamports(account.lamports),
-    rentEpoch: account.rentEpoch,
+    rentEpoch: contvertToBigInt(account.rentEpoch),
```

Comment:
What about this instead?

```suggestion
    rentEpoch: account.rentEpoch ? BigInt(account.rentEpoch) : undefined,
```

Reply:
Thanks for the suggestion, I just implemented it.